graphql/resolvers.go

```diff
@@ -1668,7 +1668,8 @@ func (r *mutationResolver) SetTaskPriority(ctx context.Context, taskID string, p
 	return apiTask, err
 }
 
-func (r *mutationResolver) SchedulePatch(ctx context.Context, patchID string, reconfigure PatchReconfigure) (*restModel.APIPatch, error) {
+func (r *mutationResolver) SchedulePatch(ctx context.Context, patchID string, reconfigure PatchReconfigure, parameters []*restModel.APIParameter) (*restModel.APIPatch, error) {
```

Comment:
Can parameters be added to PatchReconfigure instead of being a separate thing here?